Sources/MakerHelper.swift

```diff
@@ -45,16 +45,47 @@ extension Maker {
         }
 
         switch type {
-        case .top:        return convertedRect.minY
-        case .bottom:     return convertedRect.maxY
-        case .centerY:    return view.contains(self.view) ? convertedRect.height / 2 : convertedRect.midY
-        case .centerX:    return view.contains(self.view) ? convertedRect.width / 2 : convertedRect.midX
-        case .right:      return convertedRect.maxX
-        case .left:       return convertedRect.minX
-        default:          return 0
+        case .top:
+            return convertedRect.minY
+        case .bottom:
+            return convertedRect.maxY
+        case .centerY:
+            return view.contains(self.view) ? convertedRect.height / 2 : convertedRect.midY
+        case .centerX:
+            return view.contains(self.view) ? convertedRect.width / 2 : convertedRect.midX
+        case .right:
+            return convertedRect.maxX
+        case .left:
+            return convertedRect.minX
+        case .safeArea(.top):
+            if #available(iOS 11.0, *) {
+                return convertedRect.minY + view.safeAreaInsets.top
+            }
+
+            return convertedRect.minY
```

Comment:
Not sure if this logic is correct. Before iOS 11 there is `topLayoutGuide`. Can we somehow check this? 🤔

If not we should add docs for that case with detailed description

Reply:
I'm not sure it's a good idea to add any hacks to try to get layout guides here. 
I'd rather just make `nui_safeArea` available only from iOS 11+ if it helps avoid confusion.

Safe areas and layout guides are too different beasts, so it's better not to mix those.
If anyone does layout for iOS < 11, they really should do `maker.top(to: view.nui_top, inset: topLayoutGuide.length)`.